packages/utils/src/ts-eslint/Config.ts

```diff
@@ -16,8 +16,18 @@ export namespace SharedConfig {
   export type RuleEntry = RuleLevel | RuleLevelAndOptions;
   export type RulesRecord = Partial<Record<string, RuleEntry>>;
 
-  export type GlobalVariableOptionBase = 'off' | 'readonly' | 'writable';
-  export type GlobalVariableOption = GlobalVariableOptionBase | boolean;
+  export type GlobalVariableOptionBase =
+    | 'off'
+    | 'readonly'
+    | 'writable'
+    | /** @deprecated use `'writable'` */ 'writeable'
+    | /** @deprecated use `'readonly'` */ 'readable';
+  export type GlobalVariableOptionBoolean =
+    | /** @deprecated use `'writable'` */ true
+    | /** @deprecated use `'readonly'` */ false;
```

Comment:
https://eslint.org/docs/latest/use/configure/language-options#specifying-globals

![screenshot of the above docs](https://github.com/user-attachments/assets/58b5a40a-b6b1-425c-9f7a-1e18189f818f)

Reply:
_grumbles about the burden of years-long legacy support_